.gitignore

```diff
@@ -0,0 +1,4 @@
+/fpdf/
+/uploads/
+/vendor/
+credential.php
```

Comment:
Where is the credential.example.php file?

Reply:
I will upload it.

Reply:
I have uploaded it.